src/common/sandbox.c

```diff
@@ -1458,183 +1459,6 @@ sandbox_cfg_allow_openat_filename(sandbox_cfg_t **cfg, char *file)
   return 0;
 }
 
```

Comment:
So the move below of network sandbox code, is the idea to move the per-module sandbox code *into* the module or a specific sandbox module is coming?

Reply:
A sandbox module is coming. :)